docs/spec/channel/version_070.md

```diff
@@ -0,0 +1,140 @@
+  replicationFactor: 1
+```
+
+A different example for the `InMemoryChannel`:
+
+```
+apiVersion: messaging.knative.dev/v1alpha1
+  kind: InMemoryChannel
+  metadata:
+    name: my-channel
+```
+
+Each _Channel Controller_ ensures the required tasks on the backing technology are applied. In this case a Kafka topic with the desired configuration is being created, backing all messages from the channel.
+
+#### Aggregated Channelable ClusterRole
+
+Every CRD must create a corresponding ClusterRole, that will be aggregated into the `channelable-manipulator` ClusterRole. This ClusterRole must include permissions to create, read, patch, and update the CRD's custom objects. Below is an example for the `KafkaChannel`:
+
+```
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kafka-channelable-manipulator
+  labels:
+    duck.knative.dev/channelable: "true"
+# Do not use this role directly. These rules will be added to the "channelable-manipulator" role.
+rules:
+  - apiGroups:
+      - messaging.knative.dev
+    resources:
+      - kafkachannels
+      - kafkachannels/status
+    verbs:
+      - create
+      - get
+      - list
+      - watch
+      - update
+      - patch
+```
+
+Each channel MUST have the `duck.knative.dev/channelable: "true"` label on its `channelable-manipulator` CR file.
+
+#### CustomResourceDefinition per Channel
+
+For each channel implementation a `CustomResourceDefinition` is created, like:
+
+```
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+ name: kafkachannels.messaging.knative.dev
+ labels:
+    knative.dev/crd-install: "true"
+    messaging.knative.dev/subscribable: "true"
+spec:
+  group: messaging.knative.dev
+  version: v1alpha1
+  names:
+    kind: KafkaChannel
+    plural: kafkachannels
+    singular: kafkachannel
+    categories:
+    - all
+    - knative
+    - messaging
+    - channel
+    shortNames:
+    - kc
+  scope: Namespaced
+...
+```
+
+Each channel is _namespaced_ and MUST have the following:
+
+* label of `messaging.knative.dev/subscribable: "true"`
+* a `channel` category
+
+#### Broker and Triggers
+
+TODO
+
+### Data Plane
+
+The data plane describes the input and output flow of a _Channel_. All Channels exclusively communicate using CloudEvents.
+
+#### Input
+
+Every Channel must expose either an HTTP or HTTPS endpoint. It MAY expose both. The endpoint(s) MUST conform to [HTTP Transport Binding for CloudEvents - Version 0.2](https://github.com/cloudevents/spec/blob/v0.2/http-transport-binding.md). It MUST support both Binary Content mode and Structured Content mode. The HTTP(S) endpoint MAY be on any port, not just the standard 80 and 443. Channels MAY expose other, non-HTTP endpoints in addition to HTTP at their discretion (e.g. expose a gRPC endpoint to accept events).
+
+##### Generic
+
+If a Channel receives an event queueing request and is unable to parse a valid CloudEvent, then it MUST reject the request.
```

Comment:
Last time I checked there was no validation done (at least for the InMemoryChannel)

Reply:
Should be for 0.8?